monai/bundle/scripts.py

```diff
@@ -415,12 +419,22 @@ def ckpt_export(
     # convert to TorchScript model and save with meta data, config content
     net = convert_to_torchscript(model=net)
 
+    extra_files: Dict = {}
+    for i in ensure_tuple(config_file_):
+        # split the filename and directory
+        _, filename = os.path.split(i)
+        # remove extension
+        filename, _ = os.path.splitext(filename)
```

Comment:
Why not just remove this line? `filename` will contain the base name of the file which can be put directly into `extra_files`.

Reply:
Thanks for your suggestion, so you mean if the config is YAML file, we should `yaml.dumps()` to serialize it?
Now this PR always uses `json.dumps()` to serialize:
https://github.com/Project-MONAI/MONAI/pull/4159/files#diff-a715c9207f1e727d7a6ed7c6b25dfb57045d4429fc0b4eed3f5396e26ab34f80R430

Thanks.

Reply:
As discussed offline, let's save the configs without extension.

Thanks.